server/crawler/crawler.go

```diff
@@ -773,7 +777,58 @@ func extractDockerComposePackageContent(
 		"",
 		0,
 		nil,
-	), true, nil
+	)
+
+	dockerComposeEnvFilePath := path.Join(packageRepositoryMetadata.RootPath, dockerComposeEnvFilename)
+
+	envFileContentResult, _, resp, err := client.Repositories.GetContents(ctx, packageRepositoryMetadata.Owner, packageRepositoryMetadata.Name, dockerComposeEnvFilePath, repoGetContentOpts)
+	if err != nil {
+		return nil, false, stacktrace.Propagate(err, "an error occurred retrieving the docker compose env file")
+	}
+	if resp.StatusCode == http.StatusNotFound {
+		logrus.Warnf("Docker compose repository '%s' is missing an env file", packageRepositoryMetadata.Name)
+		return packageContent, true, nil
+	}
+
+	envFileContentArguments, err := extractArgumentsFromEnvFileContent(envFileContentResult)
+	if err != nil {
+		return nil, false, stacktrace.Propagate(err, "an error occurred extracting the arguments from the docker compose env file")
+	}
+	packageContent.PackageArguments = envFileContentArguments
+	
+	return packageContent, true, nil
+}
+
+func extractArgumentsFromEnvFileContent(envFileContent *github.RepositoryContent) ([]*StarlarkFunctionArgument, error) {
+	envFileContentStr, err := envFileContent.GetContent()
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "an error occurred decoding the docker compose env file")
+	}
+	envFileContentReader := strings.NewReader(envFileContentStr)
+	envVars, err := envparse.Parse(envFileContentReader)
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "an error occurred parsing the docker compose env file")
+	}
+	
+	envFileArguments := make([]*StarlarkFunctionArgument, len(envVars))
+	index := 0
+	for key, val := range envVars {
+		arg := &StarlarkFunctionArgument{
+			Name: key,
+			Description: "",
```

Comment:
An enhancement would to parse any comments found in the env file.  The dot env parsing packages don't do that so it is extra work.